src/Message/Message.tsx

```diff
@@ -40,7 +40,7 @@ export const Message: FC<MessageProps> = ({
       <Icon
         size={sizeSmall ? 24 : 32}
         render={type}
-        color={type === 'warning' ? 'pink8' : 'blue7'}
+        color={type === 'warning' ? 'primary' : 'secondary'}
```

Comment:
I've just realised that there's a small error with the migration guide. `pink8` has been used as an error color since it's basically a dark red so we should change it with `error` rather than `primary`